docs/overview.md

```diff
@@ -13,22 +16,48 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+Several solutions, such as global transaction managers, data federation engines, and HTAP systems, have similar goals, but they are limited in the following perspectives:
+
+- Global transaction managers (like Oracle MicroTx and Atomikos) are designed to run transactions across a limited set of heterogeneous databases (like only XA-compliant databases).
+- Data federation engines (like Denodo and Starburst) are designed to run analytical queries across heterogeneous databases.
+- HTAP systems (like TiDB and SingleStore) run both transactions and analytical queries only on homogeneous databases.
+
+In other words, they virtually unify databases in an incomplete way. For example, with data federation engines, users can run read-only analytical queries on a virtualized view across multiple databases. However, they often need to run update queries separately for each database.
```

Comment:
I feel that "an incomplete way" is a bit offensive, given that we list several actual product names. It would be a matter of preference, but I'd recommend moderate words, like "they virtually unify databases, but with limitations".  What do you think? cc @josh-wong

Reply:
@choplin I understand your concern. `they virtually unify databases, but with limitations` sounds a bit less aggressive, so I'd be OK with that.